src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Text/Text.test.jsx

```diff
@@ -14,6 +14,7 @@ describe('Text component', () => {
             editorContext: {
                 uilang: 'en'
             },
+            onChange: () => {},
```

Comment:
you can mock the onChange function with jest:fn() and validate if it's called or not (ex: https://github.com/Jahia/content-editor/pull/706/files#diff-90d9f6486bd3030f7dd2d7efee4feed9R92)

Reply:
done, thank you for the hint!